spec/acceptance/login_navigation.feature

```diff
@@ -1,19 +0,0 @@
-Feature: login navigation
-  Scenario: Guests
-    When I go to '/'
-    Then I should see 'Sign in with MyUSA'
-    Then I should not see 'Logout'
-
-  Scenario: Logged in users
-    When I go to '/'
-    When the user is 'liono1@some-cartoon-show.com'
-    And I login
-    Then I should see alert text 'You successfully signed in'
-    And I should see 'liono1@some-cartoon-show.com'
-    And I should see 'Logout'
-
-  Scenario: Logging out
```

Comment:
this scenario is covered by https://github.com/18F/C2/blob/master/spec/features/logout_spec.rb